examples/experimental/i3s-17-and-debug/app-debug.js

```diff
@@ -74,7 +78,9 @@ const INITIAL_DEBUG_OPTIONS_STATE = {
   // Use this to freeze loaded tiles and see on them from different perspective
   loadTiles: true,
   // Show the semantic validation warnings window
-  semanticValidator: false
+  semanticValidator: false,
+  // Use "uv-checker" texture to check UV coordinates
+  useUvChecker: false
```

Comment:
We should find a common naming style for the various debug options, e.g. `showUVDebugTexture`? I prefer shorter option names, but open to suggestions